### ISSUE TITLE
Alliance shares can be voted down 

### ISSUE THREAD

**Issue by n13** [JoinSEEDS/seeds-smart-contracts#241]
Alliance shares go into escrow

They can be voted down until they are given out, but only by the voters who voted for it in the first place  - same system we use in the eval phase

So basically alliance shares have an eval phase unitl go live

They can be voted down, and the funds are then returned from the escrow account to the funding account

Also alliance shares should be paid into escrow 100% as soon as they are approved, and not enter the eval phase like campaigns.

So basically alliances would remain in eval until golive

Not sure the best way to really implement this, but the goal is that if we find that alliances have changed their mind about Seeds, then people who voted for can remove their trust.

This means that proposals has access to call into escrow and get the funds back to the alliance fund

**Comment by IanMendozaJaimes** [comment]
Hello,  regarded to the implementation of this issue, I think we can use the current eval stage for the proposals, we can calculate how many periods are required until go live, so it can be down voted normally, and in case the go live happens in the middle of a cycle, I think another table is needed to see if that lock in the escrow contract belongs to a proposal, and if it does, then evaluate if the proposal would pass the eval stage and maybe close the proposal. What do you think?

**Comment by n13** [comment]
But we don't know how long it will take to go live - there's no set date

So I would set it to 1 year, maybe with 100, 0, 0, 0, 0, 0, (12 total numbers)

So it's all paid out instantly (but goes into escrow since it's alliance)

But then in 12 months eval

Then on go live we can set them all to done
?
Not sure what's the best way to implement, this is just some thoughts.

**Comment by IanMendozaJaimes** [comment]
Or maybe we can just ignore the payouts array and keep it in eval stage until the lock is claimed or the proposal is voted down. Then if the lock was claimed in the next onperiod the proposal will be marked as done. Maybe, I don't know.

But about the event going live in the middle of a cycle, what would happen if the proposal would fail in the next onperiod but the lock can be claimed before that, should the lock could be claimed anyway? or we evaluate the proposal to see if you can claim that

**Comment by juliolrmonteiro** [comment]
Before releasing the claim, we need to recheck the unity (not quorum), to see if proposal is still passing.
Keep proposal on evaluation until go-live.
